Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
     "config/config-builder",
     "config/generate-keypair",
     "consensus",
+    "consensus/consensus_types",
```

Comment:
Since the crate name is `consensus-types` might as well use that for the directory name